### PR TITLE
#150891917 JSEND path responses

### PR DESCRIPTION
**Pull request by wanjiku-carol**
**What does this PR do?**

Change path endpoint responses to JSEND format

**Description of the task to be completed?**

- Change path endpoint responses to JSEND format
- Update tests accordingly
- Update documentation on apiary

**How should this be manually tested?**

```$ pytest -v```

**What are the relevant pivotal tracker stories?**

[150891917](https://www.pivotaltracker.com/story/show/150891917)

### CODE REVIEW

File: api/views/paths.py [hunk 1]
```diff
@@ -22,19 +22,23 @@ def post(self):
         description = payload["description"]
         if Path.find_first(name=name):
             # prevent instances of duplicate path names
-            return {
-                "status": "fail",
-                "message": "A Path with the name {} exists.".format(name)
-                }, 409
+            response = jsonify({"status": "fail",
+            "data" : {"message":
```
Comment (andela-milesanmi) [inline api/views/paths.py]: @wanjiku-carol, Nice job with this PR. Do take note of the extra spaces though in the code e.g `"data" : {"message"` -> `"data": {"message"`.

**Comment by LarryWachira** [comment]
Rebase @wanjiku-carol

**Comment by wanjiku-carol** [comment]
@LarryWachira Rebased

File: api/views/paths.py (hunk 1 above)
Comment (wanjiku-carol) [reply api/views/paths.py]: Thanks @andela-milesanmi . Done

**Comment by LarryWachira** [comment]
LGTM

**Comment by Mnickii** [comment]
LGTM

File: api/auth.py [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 from flask import g, request, jsonify
 from flask_jwt import jwt
+from .models import db, User
```
Comment (andela-oowonikoko) [inline api/auth.py]: @wanjiku-carol There's an import from models being done on `line 7`, you can move the `db` to `line 7` and take `line 5` out.

File: migrations/README [hunk 3]
```diff
@@ -1 +1 @@
-Generic single-database configuration.
\ No newline at end of file
+Generic single-database configuration.
```
Comment (andela-oowonikoko) [inline migrations/README]: @wanjiku-carol why do we have a change in the `migrations`? 😨

File: api/auth.py (hunk 2 above)
Comment (wanjiku-carol) [reply api/auth.py]: That's actually not my change but I'm on it. Thanks @andela-oowonikoko

File: migrations/README (hunk 3 above)
Comment (wanjiku-carol) [reply migrations/README]: This is in develop too. Happened when I rebased today @andela-oowonikoko

File: api/auth.py (hunk 2 above)
Comment (andela-oowonikoko) [reply api/auth.py]: 👍

File: migrations/README (hunk 3 above)
Comment (andela-oowonikoko) [reply migrations/README]: ohk

**Comment by wanjiku-carol** [comment]
@andela-oowonikoko Illegal changes sorted

**Comment by andela-oowonikoko** [comment]
LGTM

**Comment by andela-milesanmi** [comment]
LGTM

**Pull request merged**